notify-server/src/main/java/com/lexsus/ariaddna/server/MyAdvancedEchoCreator.java

```diff
@@ -0,0 +1,23 @@
+package com.lexsus.ariaddna.server;
+
+import org.eclipse.jetty.websocket.servlet.ServletUpgradeRequest;
+import org.eclipse.jetty.websocket.servlet.ServletUpgradeResponse;
+import org.eclipse.jetty.websocket.servlet.WebSocketCreator;
+
+/**
+ * Created by Lexsus on 01.04.2017.
+ */
+public class MyAdvancedEchoCreator implements WebSocketCreator
```

Comment:
Hmm... Are you sure what this classname is matches?
May be change to AriadddnaWebSocketCreator or PushnotifyWebSocketCreator.